### PR TITLE
support creating debian for adoptopenjdk tarball

### PR DESCRIPTION
**Pull request by sravyamks**
This change creates adoptopenJDK debian package

**Testing**
Built the package on `bootstrap-18-04` system to generate `adoptopenjdk-java8-jdk_8u202b08_amd64.deb`

**Steps:**
under linux-pkg run-
`./setup.sh`
`./buildpkg adoptopenjdk`

**Verified**
```
ubuntu@ip-10-110-214-164:~/linux-pkg/packages/adoptopenjdk/tmp/artifacts$ ls
adoptopenjdk-java8-jdk_8u202b08_amd64.deb
```
**Jenkins**
http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/linux-pkg-build/job/master/job/pre-push/73/

### CODE REVIEW

File: packages/adoptopenjdk/config.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+function build() {
+        logmust cd "$WORKDIR/"
+
+        env DEB_BUILD_OPTIONS=nostrip \
+          fakeroot make-jpkg "$tarfile" <<<y
+
+        logmust mv ./*.deb artifacts/
+        rm -r ./*.tar.gz
+}
```
Comment (pzakha) [inline packages/adoptopenjdk/config.sh]: Missing a newline here

File: packages/make-jpkg/config.sh [hunk 2]
```diff
@@ -33,6 +33,7 @@ function build() {
 
 	logmust dpkg_buildpackage_default
 	logmust store_git_info
+	logmust install_pkgs "$WORKDIR/artifacts"/*.deb
```
Comment (pzakha) [inline packages/make-jpkg/config.sh]: You might want to add a comment to explain why we are installing this.

File: packages/make-jpkg/config.sh [hunk 3]
```diff
@@ -33,8 +33,10 @@ function build() {
...
 }
 
 function update_upstream() {
 	logmust update_upstream_from_git
 }
+
```
Comment (pzakha) [inline packages/make-jpkg/config.sh]: Wrong file for the new line, see https://github.com/delphix/linux-pkg/pull/18/files#diff-583cfa963a6c59c725d2a44b28d88e95R42.

Also, can you run `make check` in linux-pkg head directory to make sure that it passes the checkstyle verifications?

File: packages/make-jpkg/config.sh (hunk 2 above)
Comment (sravyamks) [reply packages/make-jpkg/config.sh]: done.

File: packages/adoptopenjdk/config.sh (hunk 1 above)
Comment (sravyamks) [reply packages/adoptopenjdk/config.sh]: fixed

File: packages/make-jpkg/config.sh (hunk 3 above)
Comment (sravyamks) [reply packages/make-jpkg/config.sh]: done.

**Pull request merged**